### ISSUE TITLE
Feature Request: Make text selectable.

### ISSUE THREAD

**Issue by AJolly** [jehy/hass-addons#9]
It'd be handy to be able to select the text to copy paste it elsewhere.

**Comment by jehy** [comment]
Hi! Do you mean stats by text? If you want to show it somewhere, you can make a screenshot.
I thought about exporting data to text, but it seems a bot excessive.

**Comment by Jeppedy** [comment]
No, just being able to see an entity in the output that I want to add to my exclusions list, for example, and be able to use my mouse to highlight the name of the entity and paste it where I want.  Right now, I can't select the text on the screen.
If you fee extra-generous, perhaps you could add a "Copy to Clipboard" function/icon next to each entity name?

**Comment by AJolly** [comment]
Exactly! For example, I'm on this page:  
![image](https://github.com/user-attachments/assets/60b9a6fd-603b-46d1-a073-8dd0103694b5)
I now want to copy the name of the top device so I can send it to this script to delete - https://community.home-assistant.io/t/6-routines-to-delete-rename-move-devices-entities-and-their-corresponding-registry-entries-data-and-metadata/755476

Or even be able to click and open the entity, or its parent device back in HA.

ps. - would be also very useful if it'd cache results - maybe with a refresh button on the top, if it hasn't been updated in xyz time.  
If you have a massive overloaded db, as I did before resetting the whole thing, then it can take 10+ minutes to generate the graphs

**Comment by Jeppedy** [comment]
My hunch is it's part of the graphing/charting library...

**Comment by jehy** [comment]
Hi. Yeah, that's a valid use case. What about a download option in JSON?
